cmd/flow-rosetta-server/main.go

```diff
@@ -62,12 +62,14 @@ func run() int {
 		flagChain string
 		flagLevel string
 		flagPort  uint16
+		retCfg    retriever.Config
```

Comment:
I don't like the inconsistency here. Please just use `flagTransactions` or something.

Regarding how to inject it, I always prefer to use _variadic function parameters_ in the package itself. That way, the package can set a sane default configuration, without caring about whether anything is explicitly set or not, see below.